client/python/tests/integration/test_no_auth.py

```diff
@@ -105,6 +105,18 @@ def create_queue(client: ArmadaClient, queue_name):
     wait_for(client, queue=queue_name)
 
 
+def test_batch_update_and_create_queues(client: ArmadaClient):
+    # Need to seperately create queue name so that it is not
```

Comment:
```suggestion
    # Need to separately create queue name so that it is not
```

Reply:
Is this not the same thing? `separately` and `separately`?